TEMPLATE.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "[insert title here]"
+permalink: "[insert path here]"
+excerpt: "Smart Contracts and Chainlink"
+whatsnext: {"[insert hyperlink title]":"insert path to hyperlink", "":""}
+metadata:
+  image:
+    0: "insert image here"
+---
+*This doc is meant to be used as a template for creating a tutorial doc. Users should emulate the structure of this document when creating their own templates. The content of this doc comes from beginners-tutorial.md.*
+
+*Prerequisites should have its own block at the beginning of the tutorial. Make sure to tell the user what knowledge they should already have and where to go if they need previous reading. Make sure to direct them to other docs pages.*
+> 👍 Prerequisites
```

Comment:
Let's standardize on "Requirements" instead of "Prerequisites" unless you have a preference here.